### DISCUSSION TITLE
What is Relay Server and how to use it?

### DISCUSSION THREAD

**Discussion by vyncusp** [rustdesk/rustdesk#1212] [General]
Hi,

Can anyone explain to me what is relay server and how do i use it?
![2022-08-08_19-27-47](https://user-images.githubusercontent.com/56208673/183417848-7c257c8d-6a62-4912-beeb-959d1a6c1a20.jpg)

**Comment by yodaphone** [comment]
the relay server is the same server but running on the port 21117

from the documentation "21117 is used for Relay services,"

**Comment by chiehw** [comment]
Fill in the IP of the relay server.

**Comment by vyncusp** [comment]
If i have more than 1 rustdesk server how do i put it?
Example :
Rustdesk server A - 123..123.123
Rustdesk server B - 321.321.321

How do i put 2 server in the rustdesk client setting?
